gnssanalysis/gn_diffaux.py

```diff
@@ -526,6 +527,7 @@ def sisre(
 def diffsp3(
     sp3_a_path, sp3_b_path, tol, log_lvl, clk_a_path, clk_b_path, nodata_to_nan=True, hlm_mode=None, plot=False, write_rac_file=False
 ):
+# Eugene: function name and description are confusing - it seems to output the SISRE instead of SP3 orbit/clock differences against the given tolerance
```

Comment:
Yes there is a bit of discrepancies between doc / name / function

Reply:
We may need a discussion before changing the names